tests/puppeteer/campaigns/utils/helpers.js

```diff
@@ -12,8 +12,8 @@ module.exports = {
   async closeBrowser(browser) {
     return browser.close();
   },
-  async setDownloadBehavior(browser) {
-    await browser._client.send('Page.setDownloadBehavior', {
+  async setDownloadBehavior(page) {
+    await page._client.send('Page.setDownloadBehavior', {
```

Comment:
It doesn't break anything ?

Reply:
Nope, just a correction, i thought we could do it for browser but it was by page (in tests, we call `setDownloadBehavior(page)` anyway)